### PR TITLE
(VANAGON-111) Update to standardized archive names

### PR DESCRIPTION
**Pull request by underscorgan**
Updated to the standard documented in VANAGON-98.

### CODE REVIEW

File: lib/vanagon/platform.rb [hunk 1]
```diff
@@ -443,21 +443,21 @@ def add_build_repository(*args)
     # @param project The Vanagon::Project to run this on
     # @return array of commands to be run
     def generate_compiled_archive(project)
-      target_directory = project.repo ? output_dir(project.repo) : output_dir
       name_and_version = "#{project.name}-#{project.version}"
+      name_and_version_and_platform = "#{name_and_version}.#{name}"
```
Comment (mwaggett) [inline lib/vanagon/platform.rb]: what is `name`?

Comment (mwaggett) [reply lib/vanagon/platform.rb]: `platform.name`?

Comment (sarameisburger) [reply lib/vanagon/platform.rb]: @mwaggett it's "the name we call it" based on the info on line 7

Comment (mwaggett) [reply lib/vanagon/platform.rb]: ooh. I don't see a `name` method though? Should it be `@name`?

Comment (underscorgan) [reply lib/vanagon/platform.rb]: the attr_accessor let's us use `name` https://github.com/underscorgan/vanagon/blob/acbe95afd215fb7901c90c732c1507c540e2c0cb/lib/vanagon/platform.rb#L8

Comment (mwaggett) [reply lib/vanagon/platform.rb]: oookay! That stuff still confuses me, so I'd love to go over it sometime.
"That stuff" being what accessors do, when to use the @ vars, etc.

**Comment by sarameisburger** [comment]
@underscorgan does this need to wait on anything to be merged in?

**Comment by underscorgan** [comment]
@sarameisburger it's good to go!

**Pull request merged**